README.md

```diff
@@ -61,6 +61,25 @@ SolidusSquare::PaymentMethod.new(
   preference_source: "square_credentials"
 ).save
 ```
+### How to set the order updated webhook
```

Comment:
I have specified the route on the sandbox/config/routes.rb file, but it doesn't work and return me the following error:

<img width="1683" alt="Screenshot 2021-11-05 at 09 34 32" src="https://user-images.githubusercontent.com/387690/140481737-d8db764f-7907-4dc1-91a6-6da6a6f52ffb.png">